cli/utils/common.go

```diff
@@ -34,3 +38,21 @@ func StringReferencesFile(path string) bool {
 	// otherwise the user also could send a directory by mistake
 	return info != nil && !info.IsDir()
 }
+
+func OpenBrowser(url string) error {
```

Comment:
Philosophical question: should we detect if the client has the UI enabled on the SO before opening the browser to emit a warning and stop the command? 
I was thinking of something to warn the user if they are trying to execute this command in a CI container.

Reply:
I think we agreed on just launching without any other checks, because there is a large number of things that can go wrong and it's difficult to check them all, at least for the initial version